large_image/tilesource/base.py

```diff
@@ -418,6 +418,9 @@ def _scaleFromUnits(
                       desiredMagnification['mm_x'])
             scaleY = (desiredMagnification['scale'] /
                       desiredMagnification['mm_y'])
+            factors = dict(nm=1000000, um=1000, mm=1, m=0.001, km=0.000001)
```

Comment:
If we define this dictionary higher up, then the `elif units in [...]:` would become `elif units in mmConversionDict:`

Reply:
Good point. Moved and renamed in [f39a716](https://github.com/girder/large_image/pull/1866/commits/f39a716d9d434c29e48fa14a827f216d313d02ec)